lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -66,10 +66,10 @@ def build_index(self, server_config, dates):
 
         return f"/{index}"
 
-    def build_index_from_metadata(self):
+    def build_index_from_metadata(self, root_index_name):
```

Comment:
You didn't add a type hint for the new parameter. However, while you're at it, it'd be nice to add the return type and a proper docstring as well! ("No good deed goes unpunished" as they say...)

```
def build_index_from_metadata(self, root_index_name: str) -> str:
```